libs/ha/ha_common_libs_k8s.py

```diff
@@ -1197,3 +1198,114 @@ def get_config_value(pod_obj, pod_list=None):
         data = yaml.safe_load(conf_fd)
 
         return True, data
+
+    @staticmethod
+    def setup_mock_monitor(node_obj):
+        """
+        :param node_obj: Object for node
+        :return: Bool
+        """
+        # TODO: Revisit once RPM for mock_health_event_publisher is available
+        LOGGER.info("Get HA pod name for setting up mock monitor")
+        ha_pod = node_obj.get_all_pods(pod_prefix=common_const.HA_POD_NAME_PREFIX)[0]
+        LOGGER.info("Install git inside ha pod %s", ha_pod)
+        try:
+            node_obj.send_k8s_cmd(operation="exec", pod=ha_pod, namespace=common_const.NAMESPACE,
+                                  command_suffix="yum install git", decode=True)
+        except IOError as error:
+            LOGGER.error("Failed to install git inside ha pod %s due to error: %s", ha_pod, error)
+            return False
+
+        LOGGER.info("Clone HA repo inside ha pod %s to get mock_health_event_publisher.py", ha_pod)
+        try:
+            node_obj.send_k8s_cmd(
+                operation="exec", pod=ha_pod, namespace=common_const.NAMESPACE,
+                command_suffix=f"test -f {common_const.MOCK_MONITOR_PATH}",
+                decode=True)
+        except IOError as error:
+            LOGGER.info("%s file doesn't exist. Error: %s", common_const.MOCK_MONITOR_PATH, error)
+            try:
+                node_obj.send_k8s_cmd(
+                    operation="exec", pod=ha_pod, namespace=common_const.NAMESPACE,
+                    command_suffix="git clone -b fault_k8s https://github.com/Seagate/cortx-ha.git",
```

Comment:
repo path can go to const

Reply:
This is temporary, so didn't create many constants for this. Once we get final RPM, this all can be removed and RPM references can be added.

Reply:
Then maybe add a TODO here to keep a tab

Reply:
TODO already added at line 1208.